lib/Logger.js

```diff
@@ -31,24 +32,31 @@ class Logger {
     winston.level = this.level;
   }
 
+  _bind() {
```

Comment:
The default linter rule objects to functions with dangling underscores. Can we just stick the bind statements at the end of the constructor? I'm assuming this will only be done once, and this also ensures that `_bind()` can't be called from any other classes.

Reply:
Yes, I'll change it.